misc/config_tools/data/whl-ipc-i5/whl-ipc-i5.xml

```diff
@@ -1,62 +1,55 @@
 	Region 2: Memory at 90000000 (64-bit, prefetchable) [size=256M]
 	00:12.0 Signal processing controller: Intel Corporation Device 9df9 (rev 30)
-	Region 0: Memory at a141e000 (64-bit, non-prefetchable) [size=4K]
+	Region 0: Memory at a131e000 (64-bit, non-prefetchable) [size=4K]
 	00:14.0 USB controller: Intel Corporation Device 9ded (rev 30)
-	Region 0: Memory at a1400000 (64-bit, non-prefetchable) [size=64K]
+	Region 0: Memory at a1300000 (64-bit, non-prefetchable) [size=64K]
 	00:14.2 RAM memory: Intel Corporation Device 9def (rev 30)
-	Region 0: Memory at a1416000 (64-bit, non-prefetchable) [disabled] [size=8K]
-	Region 2: Memory at a141d000 (64-bit, non-prefetchable) [disabled] [size=4K]
+	Region 0: Memory at a1316000 (64-bit, non-prefetchable) [disabled] [size=8K]
+	Region 2: Memory at a131d000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:16.0 Communication controller: Intel Corporation Device 9de0 (rev 30)
-	Region 0: Memory at a141c000 (64-bit, non-prefetchable) [size=4K]
+	Region 0: Memory at a131c000 (64-bit, non-prefetchable) [size=4K]
 	00:17.0 SATA controller: Intel Corporation Device 9dd3 (rev 30)
-	Region 0: Memory at a1414000 (32-bit, non-prefetchable) [size=8K]
-	Region 1: Memory at a141b000 (32-bit, non-prefetchable) [size=256]
-	Region 5: Memory at a141a000 (32-bit, non-prefetchable) [size=2K]
+	Region 0: Memory at a1314000 (32-bit, non-prefetchable) [size=8K]
+	Region 1: Memory at a131b000 (32-bit, non-prefetchable) [size=256]
+	Region 5: Memory at a131a000 (32-bit, non-prefetchable) [size=2K]
 	00:1a.0 SD Host controller: Intel Corporation Device 9dc4 (rev 30)
-	Region 0: Memory at a1419000 (64-bit, non-prefetchable) [size=4K]
-	00:1c.0 PCI bridge: Intel Corporation Device 9db8 (rev f0)
-	00:1c.4 PCI bridge: Intel Corporation Device 9dbc (rev f0)
+	Region 0: Memory at a1319000 (64-bit, non-prefetchable) [size=4K]
 	00:1d.0 PCI bridge: Intel Corporation Device 9db0 (rev f0)
 	00:1d.1 PCI bridge: Intel Corporation Device 9db1 (rev f0)
 	00:1f.0 ISA bridge: Intel Corporation Device 9d84 (rev 30)
 	00:1f.3 Audio device: Intel Corporation Device 9dc8 (rev 30)
-	Region 0: Memory at a1410000 (64-bit, non-prefetchable) [size=16K]
+	Region 0: Memory at a1310000 (64-bit, non-prefetchable) [size=16K]
 	Region 4: Memory at a1000000 (64-bit, non-prefetchable) [size=1M]
 	00:1f.4 SMBus: Intel Corporation Device 9da3 (rev 30)
-	Region 0: Memory at a1418000 (64-bit, non-prefetchable) [size=256]
+	Region 0: Memory at a1318000 (64-bit, non-prefetchable) [size=256]
 	00:1f.5 Serial bus controller [0c80]: Intel Corporation Device 9da4 (rev 30)
 	Region 0: Memory at fe010000 (32-bit, non-prefetchable) [size=4K]
-	02:00.0 Non-Volatile memory controller: Silicon Motion, Inc. Device 2263 (rev 03)
-	Region 0: Memory at a1300000 (64-bit, non-prefetchable) [size=16K]
-	03:00.0 Ethernet controller: Intel Corporation I210 Gigabit Network Connection (rev 03)
+	01:00.0 Ethernet controller: Intel Corporation I210 Gigabit Network Connection (rev 03)
```

Comment:
@Kunhui-Li 
Does it intentionally remove the nvme (02:00.0 Non-Volatile memory controller) here?
Looks like 
the line33 of https://github.com/projectacrn/acrn-hypervisor/blob/master/misc/config_tools/data/whl-ipc-i5/industry_launch_1uos_hardrt.xml 
the line 75 of https://github.com/projectacrn/acrn-hypervisor/blob/master/misc/config_tools/data/whl-ipc-i5/industry_launch_2uos.xml
The line 75 of https://github.com/projectacrn/acrn-hypervisor/blob/master/misc/config_tools/data/whl-ipc-i5/industry_launch_6uos.xml

DO passthrough this nvme device...

Shall we fix the launch scripts' xml  or shall we fix this board.xml?

Reply:
Updated board.xml for it.